moduleroot/.travis.yml

```diff
@@ -1,19 +1,46 @@
+#This file is generated by ModuleSync, do not edit.
 ---
 sudo: false
 language: ruby
-bundler_args: --without system_tests
```

Comment:
When doing that, please have a look at what's already there in the `master` configs so that we have the same groups afterwards, and do not have to make another pass.

The one thing I still need to fix in `master` gem groups is splitting `:development` off into a separate group, and add a few nice-to-haves with too heavy dependencies (guard, pry, etc...)